app/services/carrierwave_filename.rb

```
@@ -0,0 +1,7 @@
+module CarrierwaveFilename
+  def for(attachment)
+    attachment.file.path.split('/').last rescue nil
```

Would it be better to use a nil guard rather than swallowing exceptions.